internal/transform/transform.go

```diff
@@ -262,15 +262,25 @@ func isRawElement(n *astro.Node) bool {
 	if n.Type == astro.FrontmatterNode {
 		return true
 	}
+	for _, attr := range n.Attr {
+		if attr.Key == "is:raw" {
+			return true
+		}
+	}
 	rawTags := []string{"pre", "listing", "iframe", "noembed", "noframes", "math", "plaintext", "script", "style", "textarea", "title", "xmp"}
 	for _, tag := range rawTags {
 		if n.Data == tag {
 			return true
 		}
-		for _, attr := range n.Attr {
-			if attr.Key == "is:raw" {
-				return true
-			}
+	}
+	return false
+}
+
+func isWhitespaceInsensitiveElement(n *astro.Node) bool {
+	rawTags := []string{"head"}
+	for _, tag := range rawTags {
+		if n.Data == tag {
+			return true
 		}
 	}
```

Comment:
Could avoid the loop here if we're only checking for `head` elements?

Reply:
Hmm yeah I can make this an if for now and we can expand this to a loop if needed later.